--- upstream.original/src/ports/SkFontHost_FreeType_common.cpp	2021-06-16 13:17:29.000000000 +0300
+++ /Users/macmini/OsmAnd/skia/src/ports/SkFontHost_FreeType_common.cpp	2021-06-15 11:29:20.000000000 +0300
@@ -760,8 +760,12 @@
     };
 };
 
+constexpr const FT_Outline_Funcs SkFTGeometrySink::Funcs;
+
 }  // namespace
 
+
+
 bool SkScalerContext_FreeType_Base::generateGlyphPath(FT_Face face, SkPath* path) {
     SkFTGeometrySink sink{path};
     FT_Error err = FT_Outline_Decompose(&face->glyph->outline, &SkFTGeometrySink::Funcs, &sink);
