--- upstream.patched/third_party/externals/zlib/zutil.h	2026-04-16 12:59:51.031451337 +0200
+++ upstream.patched/third_party/externals/zlib/zutil.h	2026-04-16 12:17:28.614204655 +0200
@@ -145,17 +145,8 @@
 #  endif
 #endif
 
-#if defined(MACOS) || defined(TARGET_OS_MAC)
+#if defined(MACOS)
 #  define OS_CODE  7
-#  ifndef Z_SOLO
-#    if defined(__MWERKS__) && __dest_os != __be_os && __dest_os != __win32_os
-#      include <unix.h> /* for fdopen */
-#    else
-#      ifndef fdopen
-#        define fdopen(fd,mode) NULL /* No fdopen() */
-#      endif
-#    endif
-#  endif
 #endif
 
 #ifdef __acorn
