diff --git a/DEPS b/DEPS
index 4c03dbd..69a8609 100644
--- a/DEPS
+++ b/DEPS
@@ -6,7 +6,7 @@ vars = {
 
 deps = {
   "buildtools"                            : "https://chromium.googlesource.com/chromium/buildtools.git@505de88083136eefd056e5ee4ca0f01fe9b33de8",
-  "common"                                : "https://skia.googlesource.com/common.git@9737551d7a52c3db3262db5856e6bcd62c462b92",
+  "common"                                : "https://github.com/QtSkia/common.git@7c44d925ea329b093cbd259113308bcba8b7804b",
   "third_party/externals/angle2"          : "https://chromium.googlesource.com/angle/angle.git@94ec1e5f8df504269bd450d7082984685ec39dfe",
   "third_party/externals/dawn"            : "https://dawn.googlesource.com/dawn.git@604072bc2ed01018eb03bcbbf9d94042f679af63",
   "third_party/externals/dng_sdk"         : "https://android.googlesource.com/platform/external/dng_sdk.git@c8d0c9b1d16bfda56f15165d39e0ffa360a11123",
